033-OSSDatabaseMigration/Student/K8s_cheetsheet.md

```diff
@@ -0,0 +1,60 @@
+**[Home](../README.md)**
+
+# Display  all the nodes

```

Comment:
Change the headers to be `##` or even `###`... the headers overpower the text.

Reply:
Fixed